extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -905,6 +908,22 @@ public void produceLoggingCategories(BuildProducer<LogCategoryBuildItem> categor
         }
     }
 
+    @BuildStep(onlyIf = NativeBuild.class)
+    public void test(CombinedIndexBuildItem index,
+            BuildProducer<ReflectiveClassBuildItem> reflective) {
+        Collection<AnnotationInstance> annotationInstance = index.getIndex().getAnnotations(STATIC_METAMODEL);
```

Comment:
Could you make the variable plural? It would be less confusing.

Reply:
Done